frontend/src/api/k8s/dsc.ts

```diff
@@ -0,0 +1,9 @@
+import { k8sGetResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { DataScienceClusterKind } from '~/k8sTypes';
+import { DataScienceClusterModel } from '~/api/models';
+
+export const getDsc = (): Promise<DataScienceClusterKind> =>
+  k8sGetResource<DataScienceClusterKind>({
+    model: DataScienceClusterModel,
+    queryOptions: { name: 'default-dsc' },
```

Comment:
dont hard code the name, you should fetch the dsc by getting a list of them and getting the first one off that list

we do this in the backend
```tsx
export const fetchClusterStatus = async (
  fastify: KubeFastifyInstance,
): Promise<DataScienceClusterKindStatus> => {
  const result: DataScienceClusterKind | null = await fastify.kube.customObjectsApi
    .listClusterCustomObject('datasciencecluster.opendatahub.io', 'v1', 'datascienceclusters')
    .then((res) => (res.body as DataScienceClusterList).items[0])
....
```